src/theme/theme.ts

```diff
@@ -99,6 +103,209 @@ const themeOptions: ThemeOptions = {
         },
       },
     },
+    MuiCssBaseline: {
+      styleOverrides: `/* cyrillic-ext */
+      @font-face {
```

Comment:
I am lost here. Is it all necessary? What these do? What happens if these are not added to the code?
Material's docs seem shorter but I don't know in all certainty. https://mui.com/material-ui/customization/typography/#font-family

Reply:
@diego20 if we want to add a font to mosaic AND our dashboard, we need `MuiCSSBaseline`